Documentation/Signoffs/AcceleratorSystem.md

```diff
@@ -0,0 +1,51 @@
+Accelerator Pedal Position System
+--------------------------------------
+### Function of the Subsystem
+The accelerator pedal position system will take the OEM signal from the pedal by splicing into the signal wires and input them to the local microcontroller. The local microcontroller will then calculate the position based the change of the voltage from the OEM hall effect sensor.
+
+### Specifications & Constraints
```

Comment:
Where do these come from? How do you know the accuracy of a leaf pedal end to end?